doc/sphinx/source/quickstart/configuration.rst

```diff
@@ -1,28 +1,29 @@
-.. _config-user:
+.. _config:
 
 *************
 Configuration
 *************
 
 The ``esmvaltool`` command is provided by the ESMValCore package, the
-documentation on configuring ESMValCore can be found
-:ref:`here <esmvalcore:config>`.
-In particular, it is recommended to read the section on the
-:ref:`User configuration file <esmvalcore:user configuration file>`
-and the section on
+documentation on configuring ESMValCore can be found :ref:`here
```

Comment:
Having a line break in the middle of a link definition is a bit awkward to read. Has this change been introduced by some automatic formatted?

Reply:
Yeah, this was introduced by my editor which tries to limit lines to 80 characters. I don't think it's a problem, but I am happy to change it if you like.

Reply:
Is it using some plugin to do this? Maybe we could add configuration so it does not break in the middle of link text, but puts the links on their own line?

Reply:
No, it's vim's (or more specifically, the vim plugin for VSCode) hard wrapping `gq`. Don't think there's much to configure here 😄 I will change it back.

Reply:
Done in https://github.com/ESMValGroup/ESMValTool/pull/3761/commits/51ef776ea234fe9c48ac0f8a945459f14111394e